terraform/paas/migrate/manifest.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Do we need this file in the repo?

Reply:
Only while I am working on the code, once I am finished and have it all sorted I intend on deleting this.